ziti/cmd/create/config_templates/controller.yml

```diff
@@ -215,3 +215,5 @@ web:
         options: { }
       - binding: fabric
         options: { }
+      - binding: zac
```

Comment:
`binding: console`, because initialisms make poor names when a meaningful name is available and unambiguous because they require definition or explanation.